pkgs/applications/science/medicine/dcmtk/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, zlib, libtiff, libxml2, openssl, libiconv, libpng, cmake }:
+
+with stdenv.lib;
+stdenv.mkDerivation {
+  pname = "DCMTK";
+  version = "3.6.4";
+  src = fetchFromGitHub {
+      owner = "DCMTK";
+      repo = "dcmtk";
+      rev = "1967b13134308f311e6a827e616958c6a4da5bc9";
+      sha256 = "0fbx35zax8n4gayaac5bankqwzg2y2adggykbbf8lpd773jfxsp6";
+  };
+
+  buildInputs = [ cmake libpng zlib libtiff libxml2 openssl libiconv ];
+
+  meta = {
+    description = "DICOM Toolkit";
```

not really descriptive, maybe take from the long description.
```suggestion
    description = "Collection of libraries and applications implementing large parts of the DICOM standard";
```